schema/jsonschema/cerbos/audit/v1/DecisionLogEntry.schema.json

```diff
@@ -175,15 +175,15 @@
         "kind": {
           "type": "string",
           "minLength": 1,
-          "pattern": "^[[:alpha:]][[:word:]\\@\\.\\-/]*(\\:[[:alpha:]][[:word:]\\@\\.\\-/]*)*$"
+          "pattern": "^[A-Za-z][\\--9@-Z_a-z]*(:[A-Za-z][\\--9@-Z_a-z]*)*$"
```

Comment:
Another odd one 🤔

Reply:
Yeah, `syntax.Parse` simplifies character classes to the minimum set of non-overlapping ranges. No human would write that regexp (I hope 😂)